ospd_openvas/daemon.py

```diff
@@ -809,12 +809,14 @@ def update_progress(self, scan_id: str, current_host: str, msg: str):
             launched, total = msg.split('/')
         except ValueError:
             return
+
         if float(total) == 0:
             return
         elif float(total) == -1:
-            host_prog = 100
+            host_prog = -1  # Host dead
         else:
             host_prog = (float(launched) / float(total)) * 100
```

Comment:
Would be nice to use (or reuse) constants for the different progress values. Additionally calling `float` can fail e.g. `float(None)`. It would be good to catch the TypeError and do some safe error handling.